packages/content-conformance/tsconfig.json

```diff
@@ -6,7 +6,8 @@
     "moduleResolution": "NodeNext",
     "jsx": "react",
     "outDir": "./dist",
-    "declaration": true
+    "declaration": true,
+    "esModuleInterop": true,
```

Comment:
Was `esModuleInterop` necessary for one of the new dependencies?

Reply:
No it wasn't. It's an artifact of me trying to get rid of TS errors :D